python/src/aiconfig/editor/client/src/components/prompt/RunPromptButton.tsx

```diff
@@ -1,15 +1,17 @@
-import { Button, Flex, Loader } from "@mantine/core";
+import { Button, Flex, Loader, Tooltip } from "@mantine/core";
 import { IconPlayerPlayFilled, IconPlayerStop } from "@tabler/icons-react";
 import { memo } from "react";
 
 type Props = {
-  isRunning?: boolean;
   cancel: () => Promise<void>;
+  disabled: boolean;
```

Comment:
nit, make optional here and default to false
```suggestion
  disabled?: boolean;
```